mitodl/mit-open
# search pages code sharing

### What are the relevant tickets?

None

### Description (What does it do?)

This reduces repeated code between the search page and field page.

### How can this be tested?
Go to http://localhost:8063/search
Go to http://localhost:8063/c/offeror/xpro

Verify that both pages look the same
